### DISCUSSION TITLE
Navbar

### DISCUSSION THREAD

**Discussion by vitalijalbu** [heroui-inc/heroui#433] [Feature requests]
Would be fine to add a responsive navbar, something like Bootstrap

**Comment by wobsoriano** [comment]
Patiently waiting <3

**Comment by fernandoox** [comment]
We'll love this new component 🚀 ❤️

**Reply by jrgarciadev** [reply]
Of course, you will 🚀

**Reply by wobsoriano** [reply]
I want to contribute but I suck at design so...  😆

**Comment by BuchananQuantum** [comment]
Well... What's up with this navbar?

**Reply by jrgarciadev** [reply]
Hey @BuchananQuantum Here's you can see the progress https://github.com/nextui-org/nextui/commits/feat/navbar

**Reply by BuchananQuantum** [reply]
@jrgarciadev Thanks man! I've been using https://www.radix-ui.com/ Navbar Primitive, to be honest the I snagged the Navbar from the "Docs" marketing site and I've been very impressed with the code. I've learned so much about Next reviewing the "Docs App'". Holy shit man, I was doing a few things right, but a lot wrong!

**Reply by BuchananQuantum** [reply]
@jrgarciadev You may be my guy to ask for this. Transitioning over from styled components to stiches. 

How do I access {props.url} in stiches. 

const Header = styled.div<{ url?: string }>`
  height: 310px;
  background-color: ${colors.placeholder};
  ${(props) =>
    props.url &&
    css`
      background-image: url(${props.url});
      background-position: center;
      background-size: cover;
    `}

  @media(min-width: 1536px) {
    border-radius: 20px;
  }
`

**Comment by jrgarciadev** [comment]
Hey guys, hope you are well, the Navbar component was released, and I hope you enjoy it 🎉 https://twitter.com/getnextui/status/1563169730575634433

**Reply by wobsoriano** [reply]
You guys are legends.
